src/js/controller/instream-adapter.js

```diff
@@ -332,7 +332,7 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
 
         _backgroundLoadTriggered = false;
         _skipOffset = item.skipoffset || _options.skipoffset;
```

Comment:
It's good this is still `item.skipoffset || _options.skipoffset` because `options` is undefined when playing subsequent ad pods.

Reply:
This would be good to add in a code comment. @waxidiotic

Reply:
Done